clang/lib/DPCT/APINamesMath.inc

```diff
@@ -258,8 +258,8 @@ ENTRY_EMULATED("rhypotf", MapNames::getClNamespace(false, true) + "hypot")
 ENTRY_REWRITE("sincosf")
 ENTRY_EMULATED("sincospif", MapNames::getClNamespace(false, true) + "sincos")
 ENTRY_REWRITE("__sincosf")
-ENTRY_EMULATED("powf", MapNames::getClNamespace(false, true) + "pow<float>")
-ENTRY_EMULATED("__powf", MapNames::getClNamespace(false, true) + "pow<float>")
+ENTRY_EMULATED("powf", MapNames::getDpctNamespace() + "pow")
```

Comment:
If use `powf(float, double)`, can pass sycl build?

Reply:
void test() {
    float a = 2.0;
    double b =2.0;
    dpct::pow(a, b);
}

Yes, the template will be selected and cast all the type to double then call the sycl::pow.